### ISSUE TITLE
How to crop individual frame and do Annotation 

### ISSUE THREAD

**Issue by kapuranirudh** [NickeManarin/ScreenToGif#316]
Hi,

ScreenToGif tool which is an amazing tool to work on,  but I have few queries ie 

1) If I need to add an annotation to any of the frames, how to do that? 
2) If I need to explain some steps by cropping a particular frame (only 1 frame not all), how to do that?

After doing all these above steps on the frames as in we run the gif clip I need to explain the software in a better way.

PFA screenshots for the same.

Request you to please resolve my query, if possible.
![annotation](https://user-images.githubusercontent.com/37772376/39091669-b4eb5c1c-4616-11e8-837c-059b202dde57.jpg)
![how to crop](https://user-images.githubusercontent.com/37772376/39091670-b51f560c-4616-11e8-9328-9eb73beca570.jpg)

Thanks & Regards
Anirudh Kapur

**Comment by NickeManarin** [comment]
To add an annotation, you can use "Free Text" or "Caption", currently there's no way to add balloons.

Cropping/zooming for a single frame is not supported too.

**Comment by kapuranirudh** [comment]
@Nicke, thanks for your reply , yes I have tried the

**Comment by kapuranirudh** [comment]
@Nicke, yes I have already tried the caption and free text , these were the only issues I was facing , hope to see these added in future .

Thought the “Screentogif” is an amazing software.

Thanks
